cdap-hbase-compat-1.2-cdh5.7.0/pom.xml

```diff
@@ -50,6 +50,13 @@
   </dependency>
   <dependency>
     <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-hbase-compat-base</artifactId>
```

Comment:
I see ok. I will declare direct dependency on cdap-hbase-compat-base for all compat modules.